docs/rules/jsx-no-target-blank.md

```diff
@@ -18,6 +18,13 @@ There are two main options for the rule:
 * `{"enforceDynamicLinks": "always"}` enforces the rule if the href is a dynamic link (default)
 * `{"enforceDynamicLinks": "never"}` does not enforce the rule if the href is a dynamic link
 
+```json
+"react/jsx-no-target-blank": [<enabled>, { enforceDynamicLinks: <enforce> }]
+```
+
+* enabled: for enabling the rule. 0=off, 1=warn, 2=error. Defaults to 0.
+* enforce: optional string, defaults to "always"
```

Comment:
this seems redundant, since it's on lines 18-19

Reply:
Again, I'm following how it works on some other pages